pkg/user/controllers/user.go

```diff
@@ -83,23 +90,28 @@ func (uc *UserController) Create(w http.ResponseWriter, r *http.Request) {
 // @Router /tokens [post]
 // CreateToken handles the server request and calls CreateTokenInteractor
 func (uc *UserController) CreateToken(w http.ResponseWriter, r *http.Request) {
-	var user entities.User
-	err := json.NewDecoder(r.Body).Decode(&user)
+	var param CreateTokenParam
+	err := json.NewDecoder(r.Body).Decode(&param)
 	if err != nil {
 		uc.logger.Error("failed to parse request body")
 		uc.writeResponse(w, http.StatusUnprocessableEntity, nil)
 		return
 	}
 
-	token, err := uc.createTokenInteractor.Execute(user)
+	tokenType := param.Type
+	if tokenType == "" {
+		tokenType = "user"
+	}
+
+	token, err := uc.createTokenInteractor.Execute(entities.User{Email: param.Email, Password: param.Password}, tokenType)
 	if err != nil {
 		uc.logger.Errorf("failed to create user's token: %s", err)
 		der := &DetailedErrorResponse{err.Error()}
 		uc.writeResponse(w, mapErrorToStatusCode(err), der)
 		return
 	}
 
-	uc.logger.Infof("user's %s token created", user.Email)
+	uc.logger.Infof("user's %s token created", param.Email)
```

Comment:
Since we're updating this anyway :point_down: 

```suggestion
	uc.logger.Infof("token created for user %s", req.Email)
```

Reply:
Done!